Source/Core/DolphinQt/DolphinEmu.entitlements

```diff
@@ -0,0 +1,10 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
+<plist version="1.0">
+<dict>
```

Comment:
I opened #8909 which has some additional entitlements and an additional codesigning flag for hardened runtime. If this is merged, then perhaps the changes from that PR should be included here to avoid dupes?

(repost because I accidentally started a review, sorry)

Reply:
I wouldn't be opposed to adding the additional entitlements. Is there any reason the original pull request wasn't merged?

Reply:
Mostly a lack of reviewers who know macOS well, I guess.

Reply:
Good this was reposted. The lack of analog controller support is frustrating. Using Enjoyable.app as a workaround for keyboard mapping. Would enjoy being a reviewer.

Reply:
I added most of the changes in the changes for #8909 to this commit. Although, I did not add the MAP_JIT flag to the x86_64 memory allocations, because we may still need to add JITPageWrite*Execute*() to the x86 JIT for this to work right